podcasts/Bookmarks/Player/BookmarksPlayerTab.swift

```diff
@@ -0,0 +1,44 @@
+import SwiftUI
+
+struct BookmarksPlayerTab: View {
+    @EnvironmentObject var theme: Theme
+
+    var body: some View {
+        ZStack {
+            Color.blue
+
+            Text("Hello!")
+                .foregroundStyle(Color.white)
+        }
+    }
+}
+
+// MARK: - PlayerItemViewController Wrapper
+
+class BookmarksPlayerTabController: PlayerItemViewController {
```

Comment:
This is a bit of hack to use SwiftUI but still conform to this controller. I played with changing to a Protocol instead but that was a lot of extra work so I kept it simple. 

I may adjust in the future as the UI is added.